presto-spi/src/main/java/com/facebook/presto/spi/plan/TableScanNode.java

```diff
@@ -51,14 +52,16 @@ public TableScanNode(
             @JsonProperty("id") PlanNodeId id,
             @JsonProperty("table") TableHandle table,
             @JsonProperty("outputVariables") List<VariableReferenceExpression> outputVariables,
-            @JsonProperty("assignments") Map<VariableReferenceExpression, ColumnHandle> assignments)
+            @JsonProperty("assignments") Map<VariableReferenceExpression, ColumnHandle> assignments,
+            @JsonProperty("delete") boolean delete)
```

Comment:
Just noticed a test still using this interface, could you help clean that up as well? Let's put a javadoc to this constructor to show no one should use this other than serde.